src/OrchardCore.Modules/OrchardCore.Markdown/Views/Media-MarkdownBodyPart.Modal.cshtml

```diff
@@ -18,7 +15,6 @@
     </div>
 </div>
 
-<script asp-name="media" at="Foot"></script>
```

Comment:
You sure?

Reply:
Yes. Look at the following line, we depend-on="media"